solr/core/src/test/org/apache/solr/cloud/CollectionsAPIDistributedZkTest.java

```diff
@@ -70,357 +77,193 @@
-    clusterPropTest();
-    // last
-    deleteCollectionWithDownNodes();
-    addReplicaTest();
-  }
-
-  private void deleteCollectionRemovesStaleZkCollectionsNode() throws Exception {
-    
-    // we can use this client because we just want base url
-    final String baseUrl = getBaseUrl((HttpSolrClient) clients.get(0));
-    
+  public void deleteCollectionRemovesStaleZkCollectionsNode() throws Exception {
     String collectionName = "out_of_sync_collection";
     
-    List<Integer> numShardsNumReplicaList = new ArrayList<>();
-    numShardsNumReplicaList.add(2);
-    numShardsNumReplicaList.add(1);
-    
-    
     cloudClient.getZkStateReader().getZkClient().makePath(ZkStateReader.COLLECTIONS_ZKNODE + "/" + collectionName, true);
     
-    ModifiableSolrParams params = new ModifiableSolrParams();
-    params.set("action", CollectionAction.DELETE.toString());
-    params.set("name", collectionName);
-    QueryRequest request = new QueryRequest(params);
-    request.setPath("/admin/collections");
-    
-    // there are remnants of the collection in zk, should work
-    makeRequest(baseUrl, request);
-    
-    assertCollectionNotExists(collectionName, 45);
-    
-    assertFalse(cloudClient.getZkStateReader().getZkClient().exists(ZkStateReader.COLLECTIONS_ZKNODE + "/" + collectionName, true));
+    CollectionAdminResponse response = deleteCollection(collectionName)
+        .process(cloudClient);
+    assertEquals(0, response.getStatus());
 
+    waitForCollectionToDisappear(collectionName, cloudClient.getZkStateReader(), false, true, 45);
+    assertFalse(cloudClient.getZkStateReader().getZkClient().exists(ZkStateReader.COLLECTIONS_ZKNODE + "/" + collectionName, true));
   }
 
-  private void deletePartiallyCreatedCollection() throws Exception {
-    final String baseUrl = getBaseUrl((HttpSolrClient) clients.get(0));
+  @Test
+  public void deletePartiallyCreatedCollection() throws Exception {
     String collectionName = "halfdeletedcollection";
-    Create createCmd = new Create();
-    createCmd.setCoreName("halfdeletedcollection_shard1_replica1");
-    createCmd.setCollection(collectionName);
-    String dataDir = createTempDir().toFile().getAbsolutePath();
-    createCmd.setDataDir(dataDir);
-    createCmd.setNumShards(2);
-    if (secondConfigSet) {
-      createCmd.setCollectionConfigName("conf1");
-    }
 
-    makeRequest(baseUrl, createCmd);
-
-    ModifiableSolrParams params = new ModifiableSolrParams();
-    params.set("action", CollectionAction.DELETE.toString());
-    params.set("name", collectionName);
-    QueryRequest request = new QueryRequest(params);
-    request.setPath("/admin/collections");
+    Properties properties = new Properties();
+    properties.put(CoreAdminParams.DATA_DIR, createTempDir().toFile().getAbsolutePath());
+    properties.put(CoreAdminParams.CORE, "halfdeletedcollection_shard1_replica1");
+    CollectionAdminResponse response = createCollection(collectionName, configSet, 2, 1)
```

Comment:
Ah yes, I learned later that this use of properties won't actually do anything, and forgot to go back to fix this